sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -216,20 +216,42 @@ def wrapper(*args, **kwargs):
       else:
         constant_args[ix] = arg
 
+    deferred_kwarg_keys = []
```

Comment:
It might be worth an explicit test in frame_base_test for the kwargs functionality.

Reply:
Good call, adding a test caught a small mistake on L241 that would've been a pain to debug later